pkg/gadgets/trace/network/tracer/tracer.go

```diff
@@ -368,15 +369,20 @@ func (t *Tracer) Init(gadgetCtx gadgets.GadgetContext) error {
 
 func (t *Tracer) Start() error {
 	ctx := t.gadgetCtx.Context()
+	ticker := time.NewTicker(time.Second)
+	defer ticker.Stop()
+
 	for {
-		if ctx.Err() != nil {
-			return nil
-		}
-		// Pop, but we don't need the results as we're handing the events over to the callback in cbMap
-		_, err := t.Pop()
-		if err != nil {
-			t.gadgetCtx.Logger().Debugf("pop() returned with: %v", err)
+		select {
+		case <-ctx.Done():
 			return nil
+		case <-ticker.C:
+			// Pop, but we don't need the results as we're handing the events over to the callback in cbMap
```

Comment:
What is `cbMap`? That comment was there before your patch but I don't understand what it refers to.

Reply:
It refers to a callback map Mike was using during the refactoring but that we removed after a cleanup. So, the comment can be removed as it's completely out-of-date.